pymilo/pymilo_obj.py

```diff
@@ -16,22 +16,6 @@ def __init__(self, model):
         self.version = get_sklearn_version()
         self.type = get_sklearn_type(model)
 
-    def to_json(self):
```

Comment:
@AHReccese Why?

Reply:
Yeah, I removed it because it is not used in any parts of the project anymore, previously I had used it but during the updates it became useless.
@sepandhaghighi

Reply:
It's not useless we should add some tests for it.

Reply:
ok, How about this: I use the `to_json` function in the `save mathod?
@sepandhaghighi

Reply:
Good Idea 💯 
But this method has its own usage also for the end user

Reply:
ok, I will do both.